### ISSUE TITLE
Problem With BetterF3

### ISSUE THREAD

**Issue by SnowedInOwl** [Roundaround/mc-fabric-true-sight#2]
BetterF3 uses the same keybind

**Comment by a-fan9823** [comment]
Yeah I would add atleast SOME way to change the keybind cus it conflicts with better f3 even going into files would be a good fix

**Comment by Roundaround** [comment]
Unfortunately it's not something that's easy to make configurable. I've [filed an issue](https://github.com/cominixo/BetterF3/issues/155) on the BetterF3 repo asking them to change their keybind and outlined the difficulties there.

If they come back with a no I might consider changing mine to some other key, but it's not exactly something I'd like to do.

**Comment by RazerLemon** [comment]
were u able to change it?
